### DISCUSSION TITLE
Белый прямоугольник посередине окна.

### DISCUSSION THREAD

**Discussion by per-noel666** [ppy/osu#28107] [Q&A]
при прохождении любой карты посередине висит белый прямоугольник с аватаркой и никак не убирается. Что делать?

**Comment by bdach** [comment]
Use English please, and attach screenshots please.

**Comment by Loreos7** [comment]
@per-noel666 Привет, тут принято общаться на английском. И к своей проблеме нужно приложить хоть какую-либо информацию кроме текста. Дай скриншот, запиши видео, приложи логи (через настройки).
Без доп. информации могу лишь сказать, что либо проблема с диском, либо с отрисовщиком графики.
Возможно, поможет обновление графического драйвера.
